packages/editor/src/components/document-outline/index.js

```diff
@@ -151,14 +198,14 @@ export const DocumentOutline = ( {
 
 export default compose(
 	withSelect( ( select ) => {
-		const { getBlocks } = select( blockEditorStore );
-		const { getEditedPostAttribute } = select( editorStore );
+		const { getEditedPostAttribute, getEditorBlocks } =
+			select( editorStore );
 		const { getPostType } = select( coreStore );
 		const postType = getPostType( getEditedPostAttribute( 'type' ) );
 
 		return {
 			title: getEditedPostAttribute( 'title' ),
-			blocks: getBlocks(),
+			blocks: getEditorBlocks(),
```

Comment:
It does make me wonder if we should include the post title and site title blocks here at some point. Not sure if that makes sense. Perhaps then we should also include article and section boundaries I guess.